### PR TITLE
Remove variable content from weird names

### PR DESCRIPTION
**Pull request by jsiwek**
Note the `topic/jsiwek/bounded-weirds` branch is in the following repos to merge this PR completely:

* zeek
* zeek-docs
* zeek-testing
* zeek-testing-private

See commit message / NEWS for details.

### CODE REVIEW

File: src/IP.cc [hunk 1]
```diff
@@ -288,7 +288,7 @@ RecordVal* IPv6_Hdr::BuildRecordVal(VectorVal* chain) const
 			}
 
 		default:
-			reporter->Weird(fmt("unknown_mobility_type_%d", mob->ip6mob_type));
+			reporter->Weird(fmt("unknown_mobility_type", mob->ip6mob_type));
```
Comment (JustinAzoff) [inline src/IP.cc]: Should this still be fmt() like that ?

```suggestion
			reporter->Weird("unknown_mobility_type", fmt("%d", mob->ip6mob_type));
```

File: src/Sessions.cc [hunk 2]
```diff
@@ -747,7 +745,7 @@ void NetSessions::DoNextPacket(double t, const Packet* pkt, const IP_Hdr* ip_hdr
 		}
 
 	default:
-		Weird(fmt("unknown_protocol_%d", proto), pkt, encapsulation);
+		Weird("unknown_protocol", pkt, encapsulation);
```
Comment (JustinAzoff) [inline src/Sessions.cc]: Was removing the proto from addl intentional here?

File: src/IP.cc (hunk 1 above)
Comment (jsiwek) [reply src/IP.cc]: Thanks, I pushed a fix (this code path gets pre-processed out by default, so there was no compiler complaints).

File: src/Sessions.cc (hunk 2 above)
Comment (jsiwek) [reply src/Sessions.cc]: Yes, intentional -- `net_weird` and `flow_weird` have no "addl".

I'm up for adding an "addl" to those events, but it's awkward at the moment: adding an event parameter breaks user-code, which I don't like (and #262 would solve).  I also don't like the idea of adding new event names like `net_weird_addl` and `flow_weird_addl` just to add more params.

Here's a list of effected net/flow weird events:

- "unknown_mobility_type_%d" -> unknown_mobility_type
- "unknown_routing_type_%d" -> unknown_routing_type
- "unknown_protocol_%d" ->  unknown_protocol
- "unknown_gre_version_%d" -> unknown_gre_version
- "unknown_gre_protocol_%u16" -> unknown_gre_protocol

**Pull request merged**